cord19-explorer/src/main/webapp/WEB-INF/views/affResults.html

```diff
@@ -0,0 +1,69 @@
+<html layout:decorate="~{layouts/main}">
+<head>
+<title>Authority Matcher</title>
```

Comment:
I know this is just copied from other files, but let's not keep on passing the wrong app name :-D